app/services/invite_participants.rb

```diff
@@ -1,30 +1,33 @@
 # frozen_string_literal: true
 
 class InviteParticipants
-  def run(emails)
+  def self.run(emails)
     logger.info "Emailing Participants"
 
     user = nil
     emails.each do |email|
       user = User.find_by(email: email)
```

Comment:
It was already in but why is user assigned nil and then assigned again on line 9?

Reply:
So that if an exception happens the `user` is defined (and `nil`). If it's not, then the code trying to log the exception will throw the exception itself :D